docs/pages/guides/mesheryctl/working-with-mesheryctl.md

```diff
@@ -160,11 +160,11 @@ Upgrade `mesheryctl` and run Meshery on Mac or Linux with this script:
 
 ### Installing the `mesheryctl` binary
 
-Download and unzip `mesheryctl` from the [Meshery releases](https://github.com/layer5io/meshery/releases/latest) page. Add `mesheryctl` to your PATH for ease of use. Then, execute:
+Download and unzip `mesheryctl` from the [Meshery releases](https://github.com/layer5io/meshery/releases/) page. Add `mesheryctl` to your PATH for ease of use. Then, execute:
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- ./mesheryctl system start
+ mesheryctl system start
```

Comment:
```suggestion
 ./mesheryctl system start
```

The `./` is intentional in the case of the Windows deployment. Unless the user has relocated the binary, the `./` is necessary.